docs/dev/README.md

```diff
@@ -0,0 +1,31 @@
+# Chef Infra Development Documentation
+
+This directory contains a collection of useful how-to guides for both new and seasoned Chef contributors. There are many guides explaining how to build, test, and contribute to Chef Infra as well as documents describing how the many subsystems function.
+
+A good first start is our [How Chef Infra Is Built](how_chef_is_built.md) and [Chef Infra Release and Support Schedule](release_and_support_schedule.md)
+
+## How-To Guides
+
+- [Building and Installing Locally](./how_to/building_and_installing.md)
+- [Branching and Backporting Changes](./how_to/branching_and_backporting.md)
+- [Updating Dependencies](./how_to/updating_dependencies.md)
+- [Bumping Major and Minor Versions](./how_to/bumping_minor_or_major_versions.md)
+
+## Design Documents
+
+- [Client Release Cadence](./design_documents/client_release_cadence.mdÂ)
+- [Data Collection](./design_documents/data_collector.md)
+- [Action Collection](./design_documents/action_collection.md)
+- [Deprecations Within Resources](./design_documents/deprecations_in_resources.md)
+- [Resource Guard Interpreters](./design_documents/resource_guard_interpreters.md)
+- [Resource Load and Converge Methods](./design_documents/resource_load_and_converge_methods.md)
+- [Resource Property Validation Messaging](./design_documents/resource_property_validation_messaging.md)
```

Comment:
So this link - design_documents/resource_property_validation_messaging.md - does not work b/c the file name is design_documents/resource-property_validation_messaging.md). Would you like me to fix the file name in the repo?

Reply:
Fixed this file to match the other file formats with underscores. :)